libs/features/spectrum2-styles/build.mjs

```diff
@@ -0,0 +1,84 @@
+import fs from 'fs';
+import path from 'path';
+import postcss from 'postcss';
+
+const spectrumCSSPath = path.resolve('node_modules/@spectrum-css/tokens/dist/index.css');
+const miloCSSPath = path.resolve('../../deps/spectrum2-styles.css');
+
+const spectrumCSS = fs.readFileSync(spectrumCSSPath, 'utf8');
+const miloCSS = fs.readFileSync(miloCSSPath, 'utf8');
```

Comment:
I don't understand on why we read the `miloCSS`, could you elaborate on that?

In my mind: read spectrum styles -> transform properties -> output styles

Reply:
So, we read the miloCSS because this is where we are _establishing_ all the custom property names that we want to extract from spectrum styles. 

Spectrum CSS is 3,600 lines of custom properties. We're establishing only the necessary properties given to us from design, in the initial `miloCSS` file, so its not necessarily an output. This file is just being updated with Spectrum CSS values, the property names themselves are unchanging.

We read the Milo files `--s2` prefixed properties -> use them locate and retrieve the 'values' we need from Spectrum CSS -> transform the values -> update the `miloCSS` file custom properties with the new corresponding Spectrum CSS values.

Reply:
If SpectrumCSS has 3.6k lines, and miloCSS has whatever that has - what's the process of adding variables in there?

If I get it correctly:
1. miloCSS already exists and knows what properties we want
2. We compare SpectrumCSS with miloCSS update miloCSS based on the SpectrumCSS

Reply:
I was going to ask the same yesterday, but forgot. It's much clearer now that we don't need to maintain 2 files, this totally makes sense.

My other curiosity is how you manage to keep track of "static" properties/values when updating. For example `--s2-corner-radius-900: 16px; /* S2 missing */` or `--s2-font-size-1300: 60px; /* should be 58px */`. If Spectrum added that value, it would automatically update here as well and we'd just remove the comment? Or is this something better defined as Spectrum Narrative? Does Spectrum 2 plan to have those values in the final release?

Reply:
@overmyheadandbody Yes, that's correct. If it is prefixed with --s2 and has comments like `missing`, `should be xx`, it means these should be in SpectrumCSS S2 Tokens but are currently missing or incorrect. This is their beta release of S2 tokens, so it's expected. These should be included in their final release. When Spectrum updates these, they will automatically update here, and we can remove these comments. All the Spectrum Narrative (milo) static specific properties are prefixed with --sn and we manually manage these.

Reply:
@mokimo yes thats correct. The properties that milo needs currently in MiloCSS is specified by design. These should be unchanging but if there is additional properties/tokens wanted by design, yes they would need to be added in manually initially here with the correct token(property) name to be sourced from Spectrum. Yea, I can add this this somewhere in the documentation